draft-ietf-quic-applicability.md

```diff
@@ -476,8 +476,10 @@ between the endpoints on the protocol stack in use.
 Given the prevalence of the assumption in network management
 practice that a port number maps unambiguously to an application, the
 use of ports that cannot easily be mapped to a registered service name
-might lead to blocking or other interference by network elements such as
-firewalls that rely on the port number for application identification.
+might lead to blocking or other changes to the forwarding behavior by network
+elements such as firewalls that use the port number for application
+identification, often based on QoS policies or service level agreements
+(SLAs).
```

Comment:
Not sure that I understand the rationale for the last clause here.  "often" implies that you are making a claim, but I don't believe that this one can be substantiated.  Can that be dropped?

```suggestion
identification.
```

Reply:
@gorryfair proposed to add this. However, I also don't think this adds much (or might be even a bit confusing).

Reply:
The new text looks good. Martin's edit is also fine with me. ((I prefer  we omit the examples of other ways the forwarding treatment can be changed - these are not needed.))